### DISCUSSION TITLE
Contour next one retransmitting over and over?

### DISCUSSION THREAD

**Discussion by steve8x8** [NightscoutFoundation/xDrip#3516] [Q&A]
I recently found that my meter was still connected to one of the phones I use with xDrip. (I'm doing kind of round-robin with multiple transmitters and want to avoid any confusion so one transmitter - one phone ... I know about app variants.)

Since I've been using G6 sensors which need close to no calibrations, I haven't been measuring blood drops *with that particular phone* for a long time, so I wasn't surprised to hear a long series of "pling"s after the initial connection sound.

But: this repeats each time the meter is switched on, manually or by inserting a test strip, and each time I get usually 16--18, up to 24!, "pling" sounds that tell me that *a lot* of measurements is being transmitted (there's some silence after that before the device goes to sleep). I'm getting a corresponding *lot* of UploaderQueue errors because the UUID cannot be created (I suppose it's already there).

Is this a bug in the meter, or in xDrip? I'm currently running the 2024.05.22 build. - Thanks!

**Comment by Navid200** [comment]
Can you not disable Glucose meters in xDrip?
Will that not stop the connection?

**Reply by steve8x8** [reply]
It indeed would but how would I get actual readings transmitted automatically then? (Why switch on the meter at all, you might have asked?)

**Reply by Navid200** [reply]
Does your meter have an option to disable the old measurements so that it does not try to upload them any longer?
I mean does it let you clear its queue?
If not, does it have the option to completely wipe its memory?

I am only speculating here that the meter is trying to upload old readings that xDrip does not accept because of their age.  And I am also speculating that as long as your meter does not receive an acknowledgement for successful upload of every reading, every time it connects, it attempts to upload them.

**Reply by steve8x8** [reply]
No. No. No - or at least not documented. It's a Contour Next One (of the bar type).
Your explanation sounds reasonable - and having an option to just ignore (or falsely ack) those old values (maybe in devel mode) would be a way to fix it for good: I presume I'm not the only person who has an old meter and a newly setup phone...
So what about making this a feature request?

**Reply by Navid200** [reply]
I don't have any experience with Bluetooh meters.

Please make that happen again so that you hear the beeps.  Then exactly at that time, take a screenshot of the xDrip logs and post here.

**Reply by steve8x8** [reply]
Will do - for now, I'm running a different smartphone with another transmitter, and that one isn't paired with the meter.
